src/modifiers.rs

```diff
@@ -29,3 +33,26 @@ impl StatusModifier for StateFromPeopleNowPresent {
     }
 }
 
+/// This modifier adds internal version information to the output.
+pub struct LibraryVersions;
+
+impl StatusModifier for LibraryVersions {
+    fn modify(&self, status: &mut api::Status) {
+        // Add library version information
+        let api_version = api::get_version().to_string();
+        let server_version = ::get_version().to_string();
+
+        // Create version map if it doesn't exist yet
+        if status.ext_versions.is_absent() {
```

Comment:
Sorry, I wrote that in a wrong way. It does work, but we're not assigning anything. `if let` only makes sense if you want to capture the content of the case you're matching against.

Furthermore, you're matching against something (borrowing it) and then mutate the reference pointing to that value. I think in this case that would only work because enum values are `Copy`, or some similar mechanism. https://is.gd/0quEoN